.github/workflows/docker-push.yaml

```diff
@@ -0,0 +1,33 @@
+name: image-publisher
+
+on:
+  push:
+    branches: [ 'main' ]
+
+jobs:
+  docker-push:
+    name: docker-build
+    runs-on: ubuntu-latest
+    if: github.repository == 'red-hat-storage/odf-operator'
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.15'
+
+    - uses: actions/checkout@v2
+
+    - name: Login to Quay
+      uses: docker/login-action@v1
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_ROBOT_TOKEN }}
+
+    - name: Build and push docker images
+      run: |
+        export IMAGE_TAG=${GITHUB_REF#refs/heads/}
+        make docker-build docker-push
```

Comment:
I was actually thinking whether we should make these separate `run:` entries. But I guess it's fine like this.